content/client-lib-development-guide/features.textile

```diff
@@ -974,9 +984,11 @@ h4. Stats
 
 h4. ErrorInfo
 
-* @(TI1)@ Provides a generic Ably @ErrorInfo@ object that contains Ably @code@, @statusCode@ (analogous to HTTP status code) and @message@ attributes
+* @(TI1)@ Provides a generic Ably @ErrorInfo@ object that contains Ably @code@, @statusCode@ (analogous to HTTP status code), @message@ and @source@ attributes
```

Comment:
Ok, sure, happy with `cause`.  That may explain why I could not find the original issue. See https://github.com/ably/docs/issues/242!